nginx-shopify.rb

```diff
@@ -23,19 +23,9 @@ def self.nginx_modules
   depends_on "luajit-shopify"
   nginx_modules.each { |m, v| depends_on m => v }
 
-  patch do
-    url "https://gist.githubusercontent.com/csfrancis/4b6ca42b9903f4cef1e05580b2dba1d9/raw/4a64ab78362dfd876c9d4c61f2d11426a4d6de32/openresty-ssl_cert_db_yield.patch"
-    sha256 "73f29eb281e7da95f4db3b33e44d318e685fef7328e79c6c01a7393f88cef788"
-  end
-
-  patch do
-    url "https://gist.githubusercontent.com/es/ff969cad8f7461cffe285954025d0901/raw/082ae31d0e0b1b6487eb03e52b38dd215ef7a0fc/vary_header_length.patch"
-    sha256 "b5125816f824c2a8048765bff6ae71bc757424ee9387d3de759100278eeffdef"
-  end
```

Comment:
deleting this might result in some tests failing locally, can you copy https://github.com/Shopify/nginx-shopify-build/blob/master/patches/vary_header_length.patch into your gist account and refer to it here?

Reply:
I can't find a test that is testing this patch. I'd rather leave it out so we don't have to worry about updating it. If I find a test that needs it I'll come up with a better way to store this content.

Reply:
I reintroduced the patches in #68.